OsmAnd/src/net/osmand/plus/mapcontextmenu/MenuBuilder.java

```diff
@@ -336,6 +336,8 @@ public void onClick(View view) {
 		b.setTypeface(null, Typeface.BOLD);
 		b.setText(context.getResources().getString(R.string.shared_string_add_photo));
 		b.setBackgroundResource(R.drawable.btn_border_light);
+		//TODO feature under development
+		b.setVisibility(View.GONE);
```

Comment:
Here should not be gone, it should be in separate request! And this request we should test with button present